config/areas/languages/dialogs.php

```diff
@@ -249,7 +249,7 @@
 					'size'   => 'large',
 					'value'  => [
 						'key'   => $variable->key(),
-						'value' => $variable->value()
+						'value' => $variable->value() ?? ''
```

Comment:
Wondering if it should rather be fixed on line 259, where the error message also occurs. Something like

```php
App::instance()->request()->get('value', '')
````

Reply:
My first commit was that but I prefer like in PR. Both solution is good for me.

Reply:
Sorry for turning in circles, but I think maybe the actual problem is in the `LanguageVariable` class. We do allow creating one with `value = null`, so maybe it would rather be more consistent, that also `LanguageVariable::update()` allows passing `null`.